viewing_party/party.py

```diff
@@ -2,22 +2,178 @@
 
 def create_movie(title, genre, rating):
     pass
+    #make a dictionary with these keys: "title","genre","rating"
+    movie_dict = {}
+    movie_dict["title"]= title
+    movie_dict["genre"] = genre
+    movie_dict["rating"] = rating
+    
+    information = [title,genre,rating]
+    if None in information:
+        return None
+    else:
+        return movie_dict
+
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]

```

Comment:
careful here! You are re-assigning `user_data["watched"]` value, *not* appending *to* the list already inside.

What if this same user adds another movie to "watched"? It would overwrite the first movie with the new movie, instead of *appending it* to the list already declared in `user_data["watched"]`

```suggestion
    user_data["watched"].append(movie)
```